templates/plugin/dynflow_core.yml.erb

```diff
@@ -23,6 +23,13 @@
 :foreman_ssl_key: <%= scope.lookupvar("foreman_proxy::foreman_ssl_key") %>
 <% end -%>
 
+# make all remote execution jobs run asynchronously.
+<% if [nil, :undefined, :undef].include?(scope.lookupvar("foreman_proxy::plugin::dynflow::async_ssh")) -%>
+:async_ssh: false
```

Comment:
If you treat ```undef``` as ```false``` then I'd do that in ```params.pp``` and make it non-optional.

Reply:
Thanks, @ekohl updated to non-optional